tests/integration/api/set_user_id/test_set_user_id.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Tests newrelic_set_user_id() API
+*/
+
+/*EXPECT
+ok - uuid set
+*/
+
+/*EXPECT_METRICS
+[
+  "?? agent run id",
+  "??",
+  "??",
+  [
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/all"},      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/allOther"}, [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/all"},                                      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/php__FILE__"},                              [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime"},                                 [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime/php__FILE__"},                     [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Forwarding/PHP/enabled"},             [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Metrics/PHP/enabled"},                [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/api/set_user_id"},                            [1, 0, 0, 0, 0, 0]]
+  ]
+]
+*/
+
+/*EXPECT_SPAN_EVENTS
```

Comment:
Shouldn't EXPECT_TXN_TRACES and EXPECT_ANALYTICS_EVENTS be tested for presence of enduser.id attribute as well?